src/components/Search.js

```diff
@@ -97,8 +98,9 @@ export const Search = () => {
     }
   }
 
-  const onInput = ({ target }) => {
-    if (isOnline) {
+  const onInput = ({ target, isComposing }) => {
```

Comment:
Nice, this is a TIL for me: https://developer.mozilla.org/en-US/docs/Web/API/InputEvent/isComposing